src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -329,27 +349,35 @@ export const VideoTileBase: React.FC<VideoTileBaseProps> = ({
                     </ChannelHandle>
                   </Anchor>
                 ))}
-              {showMeta && (
-                <MetaContainer noMarginTop={!showChannel}>
-                  {isLoading ? (
+              <MetaContainer noMarginTop={!showChannel}>
+                {showMeta &&
+                  (isUploading ? (
+                    isLoading ? (
+                      <SpacedSkeletonLoader height={12} width={'80%'} />
```

Comment:
```suggestion
                      <SpacedSkeletonLoader height={12} width="80%" />
```

Reply:
ok, for this I'll just add https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/jsx-curly-brace-presence.md

Reply:
I mean, in another PR :P